### PR TITLE
fix(datepicker): remove position fixed

### PR DESCRIPTION
**Pull request by tarusin**
related to #3300

# PR Checklist
Before creating new PR, please take a look at checklist below to make sure that you've done everything that needs to be done before we can merge it.

 - [x] read and followed the [CONTRIBUTING.md](https://github.com/valor-software/ngx-bootstrap/blob/development/CONTRIBUTING.md) guide.
 - [x] built and tested the changes locally.
 - [x] added/updated demos.

### CODE REVIEW

File: src/datepicker/bs-datepicker.component.ts [hunk 1]
```diff
@@ -173,6 +173,10 @@ export class BsDatepickerDirective implements OnInit, OnDestroy, OnChanges {
         this.hide();
       })
     );
+
+    this._outsideScrollListener = this._renderer.listen('document', 'scroll', () => {
```
Comment (IlyaSurmay) [inline src/datepicker/bs-datepicker.component.ts]: Isn't this supposed to be in another PR? Also, this behavior should be disabled by default and this listener should only be created when corresponding input is set to true. input example `@Input() hideOnScroll = false`;

File: src/datepicker/bs-datepicker.scss [hunk 2]
```diff
@@ -511,14 +511,6 @@ bs-daterangepicker-container {
 
 /* screen size < 1024px */
 @media (max-width: 768px) {
-  bs-datepicker-container,
-  bs-daterangepicker-container {
-    position: fixed !important;
-    left: 50% !important;
```
Comment (IlyaSurmay) [inline src/datepicker/bs-datepicker.scss]: I thought we were only going to remove `position: fixed` and leave top and left

**Review by IlyaSurmay — CHANGES_REQUESTED** [review]

File: demo/src/app/components/+datepicker/datepicker-section.list.ts [hunk 3]
```diff
@@ -63,6 +64,16 @@ export const demoComponentContent: ContentSection[] = [
         `,
         outlet: DemoDatePickerCustomFormatComponent
       },
+      {
+        title: 'Manual datepicker',
```
Comment (IlyaSurmay) [inline demo/src/app/components/+datepicker/datepicker-section.list.ts]: This name doesn't reflect what demo is actually doing. Correct name is `Hide on scroll`

**Review by IlyaSurmay — CHANGES_REQUESTED** [review]

**Review by IlyaSurmay — DISMISSED** [review]

**Review by IlyaSurmay — DISMISSED** [review]

**Review by IlyaSurmay — APPROVED** [review]

**Comment by YevheniiaMazur** [comment]
Tested, looks good

**Pull request merged**